src/Game.js

```diff
@@ -1,5 +1,159 @@
 /* Manages the state of the game */
 class Game 
 {
+	constructor(board, players, endConditions)
+	{
+		this.board = board;
+		this.players = players;
+		this.endConditions = endConditions;
+		
+		/* Default turn order is alternating */
+		this.turnOrder = [
+			new Turn(this.players[0], "* *"),
```

Comment:
I don't think we pass Vectors as strings except for testing purposes..? Or did I miss something somewhere?

As for passing strings, strings are more memory-efficient and human-readable than a data structure, and I don't feel like it's any more effort to parse them than to parse a data structure. If we were in a more confining language I wouldn't use strings, but JS allows access to an uninitialized attribute (e.g., {}.thingie is undefined, not an error), so there's no way to guarantee we receive a valid input. Basically, I think this is an equal or better solution because JS. Willing to have my mind changed if you have a reason other than "coding standard."

Reply:
Other comment about strings: more adaptable when you have varying input. Notably, where you commented about EndCondition deconstructing into an array of variables, not all EndConditions need the same information, so their configuration is of varying length. Ergo, it can't be deconstructed uniformly. String parsing, by its nature, doesn't run into this issue (because it allows you to code in a state machine).